modules/caddyhttp/server.go

```diff
@@ -215,8 +215,8 @@ type Server struct {
 	// We recommend for most users to simply let Caddy use the
 	// default settings.
 	//
-	// Default: `[h1 h2 h3]`
-	Protocols []string `json:"protocols,omitempty"`
+	// Default: `[[h1 h2 h3]...]`
+	Protocols [][]string `json:"protocols,omitempty"`
```

Comment:
This is a breaking change. It breaks existing configuration.

Reply:
This can't be changed in this way; but do we need to at all? Protocols are more of a server configuration than having to do with the individual listeners.